plugins/node/opentelemetry-instrumentation-pino/src/types.ts

```diff
@@ -19,7 +19,11 @@ import { InstrumentationConfig } from '@opentelemetry/instrumentation';
 import type { pino } from 'pino';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type LogHookFunction = (span: Span, record: Record<string, any>) => void;
+export type LogHookFunction = (
+  span: Span,
+  record: Record<string, any>,
+  level?: number
```

Comment:
I think we should mention somewhere (e.g. here or in readme) that level is only provided for pino >7.9.

Reply:
Done